loadtesting/proxy/tests/erc20.py

```diff
@@ -55,13 +55,37 @@ def task_send_erc20(self):
             )
             del contracts[contract_address]
             return
-        recipient = random.choice(self.user.environment.shared.accounts)
+        recipient = self.web3_client.create_account()
         self.log.info(
             f"Send `{ERC20_CONTRACT_NAME}` tokens from contract {str(contract.address)[-8:]} to {str(recipient.address)[-8:]}."
         )
+
+        sender_balance_before = contract.functions.balanceOf(
+            self.account.address).call()
+        recipient_balance_before = contract.functions.balanceOf(
+            recipient.address).call()
+
+        amount = random.randint(1, int(sender_balance_before / 4))
         tx_receipt = self.web3_client.send_erc20(
-            self.account, recipient, 1, contract.address, abi=contract.abi
```

Comment:
in proxy methods amount is not important but very important rarely get tokens from faucet